### PR TITLE
Feat/pure pursuit

### PR DESCRIPTION
**Pull request by noahzemlin**
Adds a pure pursuit python class.
Adds a navigation node to use pure pursuit to follow a path

### CODE REVIEW

**Comment by noahzemlin** [comment]
@ddevoreou @TheMasterKelvin77 I got pure pursuit working in Python, so you might be interested in looking at this. There is the nav node which handles the navigation and robot command but the pure_pursuit.py is a complete pure pursuit class

File: igvc_ws/src/igvc_msgs/msg/path.msg [hunk 1]
```diff
@@ -0,0 +1 @@
+igvc_msgs/gps[] points
```
Comment (jkleiber) [inline igvc_ws/src/igvc_msgs/msg/path.msg]: Isn't there a standard Point msg?

Comment (noahzemlin) [reply igvc_ws/src/igvc_msgs/msg/path.msg]: Yea I agree with this. The path msg should be more diverse. The igvc_nav_node for now requires GPS coordinates (x=lat, y=long) as it uses the GPS coordinates from the EKF filter_output

**Review by jkleiber — APPROVED** [review]
LGTM

**Pull request merged**